src/index.css

```
@@ -808,6 +808,96 @@ body {
   }
 }
 
+/* ── Tree showcase animations ── */
+
+/* Stage-out: old image scales down + blurs + rotates slightly */
+@keyframes treeStageOut {
```

Stylelint (17.6.0)

[error] Expected keyframe name "treeStageOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)